### PR TITLE
EXP-620 [Configurações] Inserir banner de informação sobre chave API 

### PR DESCRIPTION
**Pull request by LucasRichter**
## Contexto
Inserir o banner de alerta na tela de configurações > Chaves API, dizendo para os clientes tomarem cuidado ao  compartilhar as chaves de API do Pagar.me

## Issues linkadas
[[Configurações] Inserir banner de informação sobre chave API](https://mundipagg.atlassian.net/secure/RapidBoard.jspa?rapidView=185&projectKey=EXP&modal=detail&selectedIssue=EXP-620&assignee=603657b8d3fc7c006805200f)

## Como era
image

## Como ficou
image

## Como testar?
- Ir em "Configurações > Chaves API" e ver se a mensagem aparece

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c7741dc1a9aa1dbc481b2f7c7db16898db444dee
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c7741dc1a9aa1dbc481b2f7c7db16898db444dee
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c7741dc1a9aa1dbc481b2f7c7db16898db444dee
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c7741dc1a9aa1dbc481b2f7c7db16898db444dee
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Review by diegodsgarcia — COMMENTED** [review]
Acho que vale desativar o warning do dangerouslySetInnerHTML no InfoMessage, pois como ele não estava sendo utilizado em lugar nenhum, não tava aparecendo:
image
```diff
--- a/packages/pilot/src/components/InfoMessage/index.js
+++ b/packages/pilot/src/components/InfoMessage/index.js
@@ -4,6 +4,7 @@ import { Alert } from 'former-kit'
 import PropTypes from 'prop-types'
 import style from './style.css'
 
+/* eslint-disable react/no-danger */
 const InfoMessage = ({
   message,
 }) => (

```
De resto ta show!!! :clap: :clap: :rocket:

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c7741dc1a9aa1dbc481b2f7c7db16898db444dee
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 0dc6c58eea0df1254d5fb6673feeac41d5b5cf16
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c7741dc1a9aa1dbc481b2f7c7db16898db444dee
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 0dc6c58eea0df1254d5fb6673feeac41d5b5cf16
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c7741dc1a9aa1dbc481b2f7c7db16898db444dee
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 0dc6c58eea0df1254d5fb6673feeac41d5b5cf16
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c7741dc1a9aa1dbc481b2f7c7db16898db444dee
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 0dc6c58eea0df1254d5fb6673feeac41d5b5cf16
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c7741dc1a9aa1dbc481b2f7c7db16898db444dee
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 0dc6c58eea0df1254d5fb6673feeac41d5b5cf16
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c7741dc1a9aa1dbc481b2f7c7db16898db444dee
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 0dc6c58eea0df1254d5fb6673feeac41d5b5cf16
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c7741dc1a9aa1dbc481b2f7c7db16898db444dee
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 0dc6c58eea0df1254d5fb6673feeac41d5b5cf16
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c7741dc1a9aa1dbc481b2f7c7db16898db444dee
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 0dc6c58eea0df1254d5fb6673feeac41d5b5cf16
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Review by diegodsgarcia — COMMENTED** [review]
No histórico dos commit faltou uma coisinha. Nós costumamos a fazer o rebase enquanto estamos desenvolvendo um PR pra assim facilitar a visualização do histórico de commits. Só realizamos merge quando finalizamos o PR com outra branch.
Se vocês dar um `git pull origin master --rebase` e resolve os conflitos já vai fica bem melhor o histórico. Dai é só dar um `git push origin feature/add-api-key-message --force-with-lease`.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d84ffb2c35ac789b0dfcb056d5880097294561af
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: dc70c8f636ff0ec2cd43fc7d58c33f3b40a833e0
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d84ffb2c35ac789b0dfcb056d5880097294561af
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: dc70c8f636ff0ec2cd43fc7d58c33f3b40a833e0
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d84ffb2c35ac789b0dfcb056d5880097294561af
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: dc70c8f636ff0ec2cd43fc7d58c33f3b40a833e0
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d84ffb2c35ac789b0dfcb056d5880097294561af
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: dc70c8f636ff0ec2cd43fc7d58c33f3b40a833e0
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

File: packages/pilot/src/containers/Settings/Company/ApiKeysTab/ApiKey/index.js [hunk 1]
```diff
@@ -20,6 +21,22 @@ const ApiKeyContainer = ({
...
+        <Row>
+          <Col
+            palm={12}
+            tablet={12}
+            desk={12}
+            tv={12}
+          >
+            <InfoMessage>
+              <p className={style.message}>
+                {t('api_key_info_1')}
```
Comment (Hrfreire) [inline packages/pilot/src/containers/Settings/Company/ApiKeysTab/ApiKey/index.js:33]: Essas duas chaves, `api_key_info_1` e `api_key_info_2`, não existem no arquivo de tradução, somente `api_key_info`.

image

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d84ffb2c35ac789b0dfcb056d5880097294561af
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: dc70c8f636ff0ec2cd43fc7d58c33f3b40a833e0
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 160d8d6a9a50c46f2bf4ccab12ec67e0e929347f
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d84ffb2c35ac789b0dfcb056d5880097294561af
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: dc70c8f636ff0ec2cd43fc7d58c33f3b40a833e0
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 160d8d6a9a50c46f2bf4ccab12ec67e0e929347f
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d84ffb2c35ac789b0dfcb056d5880097294561af
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: dc70c8f636ff0ec2cd43fc7d58c33f3b40a833e0
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 160d8d6a9a50c46f2bf4ccab12ec67e0e929347f
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: d84ffb2c35ac789b0dfcb056d5880097294561af
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: dc70c8f636ff0ec2cd43fc7d58c33f3b40a833e0
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

 * Commit: 160d8d6a9a50c46f2bf4ccab12ec67e0e929347f
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Review by diegodsgarcia — APPROVED** [review]

**Review by Hrfreire — APPROVED** [review]

**Pull request merged**